extern/storage-sealing/fsm.go

```diff
@@ -348,7 +348,9 @@ func (m *Sealing) restartSectors(ctx context.Context) error {
 				timer := time.NewTimer(cfg.WaitDealsDelay)
 				go func() {
 					<-timer.C
-					m.StartPacking(sector.SectorNumber)
+					if err := m.StartPacking(sector.SectorNumber); err != nil {
+						log.Errorf("starting sector %d: %+v", sector.SectorNumber, err)
```

Comment:
Is there an issue about this? This seems problematic.

Reply:
In theory this should only fail if something else called StartPacking on that sector.

In practice we need to rewrite a bunch of code in this general area to be more reliable anyways